packages/analyzer/CHANGELOG.md

```diff
@@ -5,7 +5,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
-<!-- ## Unreleased -->
+## Unreleased
+* Make `HtmlDocument::stringify()` faster by only cloning the ast and stringifing
```

Comment:
We've been using # to denote instance fields. So `HtmlDocument#stringify()`

Reply:
Ok, will do